Bricks/net/tcp/impl/posix.h

```diff
@@ -440,7 +467,23 @@ class Socket final : public SocketHandle {
     }
 #endif
     BRICKS_NET_LOG("S%05d accept() : OK, FD = %d.\n", static_cast<SOCKET>(socket), handle);
-    return Connection(SocketHandle::FromHandle(handle));
+
+    sockaddr_in addr_serv;
+    auto addr_serv_length = sizeof(sockaddr_in);
+#ifndef CURRENT_WINDOWS
+    if (::getsockname(handle,
+                      reinterpret_cast<struct sockaddr*>(&addr_serv),
+                      reinterpret_cast<socklen_t*>(&addr_serv_length)) == -1) {
```

Comment:
Why not `!= 0` here, same as in Windows? The `man` page says `... On success, zero is returned.`.